openpype/plugins/publish/collect_sequence_frame_data.py

```diff
@@ -50,4 +50,7 @@ def get_frame_data_from_repre_sequence(self, instance):
             return {
                 "frameStart": repres_frames[0],
                 "frameEnd": repres_frames[-1],
+                "handleStart": 0,
+                "handleEnd": 0,
+                "fps": instance.context.data["projectEntity"]["data"]["fps"]
```

Comment:
Shouldn't that be taken from `"assetEntity"`?

Reply:
Not in this plugin. We are having dedicated plugin which is optional `CollectFrameDataFromAssetEntity`.

Reply:
Note that due to this PR that particular plug-in will not do anything since it'll only handle missing keys in `instance.data` and it runs after this collector, right?

Reply:
> Not in this plugin. We are having dedicated plugin which is optional CollectFrameDataFromAssetEntity.

As BigRoy mentioned, if `"fps"` is filled here, `CollectFrameDataFromAssetEntity` won't do anything. At the same time `CollectFrameDataFromAssetEntity` can't override the value if is enabled, because the values could be filled from dedicated host collector. So it should be filled from `"assetEntity"` or not filled.

Reply:
If it is not added here and the optional plugin `CollectFrameDataFromAssetEntity` is disabled then it is crashing on ExctractReview, since it is missing `fps` at the instance. 

>  values could be filled from dedicated host collector
We are solving only TrayPublisher here since it is only host added into hosts filter.

Reply:
> If it is not added here and the optional plugin CollectFrameDataFromAssetEntity is disabled then it is crashing on ExctractReview, since it is missing fps at the instance.

I understand it might fail for that scenario and thus there's a bug we want to fix but this PR will still be breaking `CollectFrameDataFromAssetEntity` even if it were enabled. Yes, it'll allow to publish but it'll be publishing the wrong data. Right?

Reply:
Yeah, I see the issue here finally. I had changed it it